src/ui/splat-list.ts

```diff
@@ -1,4 +1,7 @@
-import { Container, Label, Element } from 'pcui';
+import { Container, Label, Element as PcuiElement } from 'pcui';
+import { Events } from '../events';
+import { Splat } from '../splat';
+import { Element, ElementType } from '../element';
```

Comment:
Any chance of renaming the app's `Element` class since it clashes with PCUI?

Reply:
Could do. What would you recommend? Perhaps "Entity"? :)

Reply:
LOL. That's taken too!

Reply:
Section, piece, segment, bit, feature, detail, unit, part

Reply:
NativeElement